src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -496,7 +496,7 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_tanh)
 
 // arcsinh
 MXNET_OPERATOR_REGISTER_UNARY(arcsinh)
-.describe(R"code(Inverse hyperbolic sine, element-wise.
+.describe(R"code(Returns element-wise inverse hyperbolic sine value of the input.
```

Comment:
"Returns the element-wise inverse hyperbolic sine of the input array, computed element-wise."